src/frontend/src/components/public/PIAFormTabs/Next_Steps/nextStepsDelegatedFlow.tsx

```diff
@@ -0,0 +1,27 @@
+import messages from './helper/messages';
+
+const NextStepsDelegatedFlow = () => {
+  return (
+    <>
```

Comment:
FYI, you don't need to use fragments when you have a single top-level element already. Not really important here, nothing will break, but just thought you should know.

Reply:
Yeah, vue2 had this one level per render and now they have removed this in vue3. React still needs one so it is good practice incase I forget why things are breaking. Vue3 might spoil me.